src/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard.test.ts

```diff
@@ -109,7 +109,6 @@ describe('AzureBillingProjectWizard', () => {
     testStepActive(4);
     verifyCreateBillingProjectDisabled();
     await nameBillingProject(billingProjectName);
-    expect(await axe(renderResult.container)).toHaveNoViolations();
```

Comment:
We already test accessibility at this same point in a similar test (slight variation of which option is selected, but that shouldn't matter).